lib/active_admin/resource_controller/streaming.rb

```diff
@@ -8,7 +8,7 @@ class ResourceController < BaseController
     #
     module Streaming
 
-      def index
+      def index(arg=nil)
```

Comment:
Another way of specifying that it accepts arguments without having to name them would be:

```suggestion
      def index(*)
```